hypervisor/include/arch/x86/guest/io_emul.h

```diff
@@ -23,6 +23,131 @@
 #define RTC_PIO_IDX		(PM1B_CNT_PIO_IDX + 1U)
 #define EMUL_PIO_IDX_MAX	(RTC_PIO_IDX + 1U)
 
+#define IO_ATTR_R               0U
```

Comment:
What is the main reason to move these code from common to be X86 specific?  To me, they are common, though they may be used in X86 specific instruction emulation side.

Reply:
the file name in common dir is now io_req.c and it mainly handle io request between DM & guest. Meanwhile, io_emul.c in arch dir is handling io emulation in HV & DM,  so actually io_req.c is a sub-stream of io_emul.c. For these structures, yes they should be common, but it's not suitable to put in io_req.h I think.